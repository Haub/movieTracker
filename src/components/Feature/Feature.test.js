import React from 'react';
import { shallow } from 'enzyme';
import { Feature} from './index.js';

describe('FEATURE', () => {
  it('should match the snapshot', () => {
    const mockMovies = [];
    const wrapper = shallow(<Feature movies={mockMovies} />)
    expect(wrapper).toMatchSnapshot();
  })

  it('should have initial state of play set to false', () => {
    const mockMovies = [];    
    const wrapper = shallow(<Feature movies={mockMovies} />) 
    expect(wrapper.state().play).toEqual(false);
  })
   
})

describe('mapStateToProps', () => {

})

describe('mapDispatchToProps', () => {

})