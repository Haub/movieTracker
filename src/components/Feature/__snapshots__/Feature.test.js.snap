// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FEATURE should match the snapshot 1`] = `<div />`;
